pom.xml

```diff
@@ -786,6 +786,14 @@
         <version>3.1.0</version>
       </dependency>
 
+      <!-- Annotations for IDE integration and analysis -->
+      <dependency>
+        <groupId>org.jetbrains</groupId>
+        <artifactId>annotations</artifactId>
```

Comment:
Since 4.1 has Java 6 compatibility, I think we need to use the `annotations-java5` variant. The one pulled in here expects Java 8 or better.